JS/Lesson34/gulcin/recipe-app/src/App.css

```diff
@@ -1,6 +1,19 @@
+@import url('https://fonts.googleapis.com/css2?family=Titillium+Web:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700&display=swap');
+
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+
+  font-family: "Titillium Web", sans-serif;
+  font-size: 16px;
+}
+
```

Comment:
I think classes grey, yellow, white are not used in your application, remove them. Every line of code is a liability, it's always better to remove unused code. :)